File: src/main/java/arcana/server/worldgen/ModFeatures.java
package arcana.server.worldgen;

import arcana.common.aspects.Aspects;
import net.minecraft.world.gen.feature.NoFeatureConfig;
import net.minecraft.world.gen.feature.StructureFeature;
import net.minecraft.world.gen.feature.structure.Structure;

public class ModFeatures {
    public static Tower towerAir = new Tower(Aspects.AIR);
    public static Tower towerWater = new Tower(Aspects.WATER);
    public static Tower towerEarth = new Tower(Aspects.EARTH);
    public static Tower towerFire = new Tower(Aspects.FIRE);
    public static Tower towerOrder = new Tower(Aspects.ORDER);
    public static Tower towerChaos = new Tower(Aspects.CHAOS);

}

File: src/main/kotlin/arcana/server/worldgen/ModFeatures.kt
package arcana.server.worldgen

import arcana.common.aspects.Aspects

object ModFeatures {
    val towerAir = Tower(Aspects.AIR)
    val towerWater = Tower(Aspects.WATER)
    val towerEarth = Tower(Aspects.EARTH)
    val towerFire = Tower(Aspects.FIRE)
    val towerOrder = Tower(Aspects.ORDER)
    val towerChaos = Tower(Aspects.CHAOS)
}